src/screens/Ticketing/Purchase/Overview/index.tsx

```diff
@@ -87,11 +35,13 @@ const PurchaseOverview: React.FC<OverviewProps> = ({
     .filter(productIsSellableInApp)
     .filter((product) => product.type === productType);
 
-  const [preassignedFareProduct, setPreassignedFareProduct] = useState(
-    params.preassignedFareProduct ?? selectableProducts[0],
-  );
+  const preassignedFareProduct =
+    params.preassignedFareProduct ?? selectableProducts[0];
```

Comment:
Typescript gives this the type `PreassignedFareProduct`. Is it not more accurate with `PreassignedFareProduct | undefined`? It works ok now, but the app was crashing for me while the react navigation updates was ongoing, because preassignedFareProduct was undefined.